weblate_musicbrainz_format_check/checks.py

```diff
@@ -17,7 +17,7 @@
             [_A-Za-z][_0-9A-Za-z]*      # identifier
         )
         (?:                             # optional replacement text
-            :                           # ':' separator
+            :?                          # ':' separator (optional)
```

Comment:
This commit https://github.com/metabrainz/musicbrainz-weblate-checks/pull/1/commits/9c23a599dc886943e4a78c8d9d48ed89f9c1ee08#diff-d631b9a0de3eaccfeff32daa9a42c110d98b8f867dc1465918c3ad5cad836fabL20-R20 would allow random characters to occur before the `|` separator. I’m replacing it in commmit https://github.com/metabrainz/musicbrainz-weblate-checks/pull/1/commits/8f91f1913b05213fc8154ccc4b9c64f14bc6ae7e and reverting in commit https://github.com/metabrainz/musicbrainz-weblate-checks/pull/1/commits/e62fa57d9777b62500b755c1ff50a4374836192d (to avoid a force push on your pull request).